app/src/main/res/values-de/strings.xml

```diff
@@ -54,15 +54,15 @@
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- Short description has to have less than 80 chars -->
-    <string name="app_short_description">Ein schlichter Musik-Player mit anpassbarem Widget, ganz ohne Werbung.</string>
+    <string name="app_short_description">Ein einfacher Musik-Player mit anpassbarem Widget, ganz ohne Werbung.</string>
     <string name="app_long_description">
-        Ein schlichter Musik-Player, der ganz einfach über die Benachrichtigungsleiste, das Homescreen Widget oder die Tasten an deinem Headset gesteuert werden kann. Du kannst alle Songs nach Titel oder Künstler sortieren und den Equalizer anpassen. Das Filtern nach Künstler oder Album wird noch nicht unterstützt, bleib dran!
+        Ein einfacher Musik-Player, der ganz einfach über die Benachrichtigungsleiste, das Homescreen Widget oder die Tasten an deinem Headset gesteuert werden kann. Du kannst alle Songs nach Titel oder Künstler sortieren und den Equalizer anpassen. Das Filtern nach Künstler oder Album wird noch nicht unterstützt, bleib dran!
         
-        Die App beinhaltet ein Widget mit anpassbarer Textfarbe, Transparenz und Hintergrundfarbe.
+        Die Anwendung beinhaltet ein Widget mit anpassbarer Textfarbe, Transparenz und Hintergrundfarbe.
         
         Beinhaltet keine Werbung oder unnötige Berechtigungen. Sie ist komplett Open Source, alle verwendeten Farben sind anpassbar.
 
-        Diese App ist nur eine aus einer größeren Serie von schlichten Apps. Der Rest davon findet sich auf http://www.simplemobiletools.com
+        Diese Anwendung ist nur eine aus einer größeren Serie. Der Rest davon findet sich auf http://www.simplemobiletools.com
```

Comment:
are you sure Anwendung is a better word? Translators dont really translate it like that.

Reply:
As App is short for Application, I think "Anwendung" fits better. But you can use both. App is just an anglicism.

Reply:
ye I want to keep it, I like those current german translations